apps/console/src/features/applications/components/forms/inbound-oidc-form.tsx

```diff
@@ -1243,6 +1246,14 @@ export const InboundOIDCForm: FunctionComponent<InboundOIDCFormPropsInterface> =
         }, [ initialValues ]
     );
 
+    useEffect(
+        () => {
+            if (initialValues?.publicClient) {
+                setPublicClient(initialValues.publicClient);
+            }
```

Comment:
Do we need this set method to be inside an if block ?
```suggestion
           setPublicClient(initialValues?.publicClient);
```